packages/pigment-css-utils/src/utils/parseExpression.ts

```diff
@@ -0,0 +1,16 @@
+import { parseExpression, ParserOptions } from '@babel/parser';
+import { ArrayExpression } from '@babel/types';
+
+/**
+ * Parses array expression string to AST for locating sourcemap points.
+ */
+export function parseArray(
```

Comment:
Should we rename the file to `parseArray.ts`, or at least to `parseExpressions.ts` (plural) if we plan to add more things.